### PR TITLE
Example Docs

### PR DESCRIPTION
**Pull request by astitva1905**

### CODE REVIEW

File: examples/mass_spring_damper.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+The mass spring damper study comprises two mass spring dampers and demonstrates how the sucessive substitution technique can be used to ensure that a co-simulation is stable. More information about successive substitution and other co-simulation stabilization techniques, please see [this paper](https://arxiv.org/pdf/1702.00686v1).
+
+
+# Example Diagram
+
+![Mass Spring Damper System](mass-spring-damper_multibody_system.png)
+
+# Example Structure
+
+There are two simulators included in the study, each representing a mass spring damper system. The first simulator calculates the mass displacement and speed of *m1* for a given force *Fk* acting on mass *m1*. The second simulator calculates force *Fk* given a displacement and speed of mass *m1*. By coupling these simulators, the evolution of the position of the two masses is computed.
```
Comment (prasadtalasila) [inline examples/mass_spring_damper.md:13]: please use underscores.
you can use math expressions. For example, see https://into-cps-association.github.io/DTaaS/development/user/digital-twins/create.html and its markdown file

File: examples/mass_spring_structure.png
Comment (prasadtalasila) [inline examples/mass_spring_structure.png:1]: please add the names of FMUs like

```txt
MassSpringDamper1
        (FMU1)
```
in the first box. Similarly to the second box.

File: examples/mass_spring_damper.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+```
+lifecycle/create
+```
+
+### Execute
+
+```
+lifecycle/execute
+```
```
Comment (prasadtalasila) [inline examples/mass_spring_damper.md:55]: mention about the location of output by showing the output of

```sh
ls /workspace/examples/data/mass-spring-damper/output
```

File: examples/mass_spring_structure.png
Comment (prasadtalasila) [reply examples/mass_spring_structure.png:1]: Please export the figure without the name of the example

**Review by prasadtalasila — COMMENTED** [review]
The contents of this directory should be in `docs/user/examples`. Do remember to link these pages in `mkdocs.yml` and `mkdocs-github.yml`

File: docs/user/examples/index.md [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+cd /root/Desktop/
+wget -P workspace https://github.com/INTO-CPS-Association/DTaaS-examples/archive/refs/heads/main.zip
+unzip workspace/main.zip -d workspace
+mv workspace/DTaaS-examples-main workspace/examples
+rm workspace/main.zip
+```
+
+The code for all examples is now setup in the workspace/examples folder and is ready for creation and execution. 
 
-The examples are hosted in
```
Comment (prasadtalasila) [inline docs/user/examples/index.md:3]: Please keep this text

File: docs/user/examples/mass-spring-damper_multibody_system.png
Comment (prasadtalasila) [inline docs/user/examples/mass-spring-damper_multibody_system.png:1]: perhaps all the three mass-spring-damper files can be put in **mass-spring-damper** directory.

File: docs/user/examples/mass_spring_damper.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
+# Mass Spring Damper
+
+## Overview
```
Comment (prasadtalasila) [inline docs/user/examples/mass_spring_damper.md:3]: Please fix all the markdownlint issues

File: docs/user/examples/mass_spring_damper.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+```
+lifecycle/execute
+```
+
+## Examine the results
+
+The results can be found in the _workspace/examples/data/mass-spring-damper/output directory_.
+
+You can also view run logs in the _workspace/examples/digital_twins/mass-spring-damper_.
+
```
[the remark this answers is not in the document]
Comment (prasadtalasila) [reply docs/user/examples/mass_spring_damper.md:68]: The logs are in __workspace/examples/data/mass-spring-damper/output_ directory.

File: docs/user/examples/mass_spring_structure.png
Comment (prasadtalasila) [inline docs/user/examples/mass_spring_structure.png:1]: please increase the resolution of the image.
Also add the examples.drawio file to _user/examples_ directory.

File: docs/user/examples/water_tank_FI/README.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+# Example Structure
+
+![Water Tank Structure](watertank_structure.png)
+
+## Configuration of assets
+
+This example uses two models and one tool. The specific assets used are:
+
+| Asset Type | Names of Assets | Visibility | Reuse in Other Examples |
+|:---|:---|:---|:---|
```
Comment (prasadtalasila) [inline docs/user/examples/water_tank_FI/README.md:22]: The two models used are:

watertankcontroller-c.fmu",
singlewatertank-20sim.fmu"

File: mkdocs.yml [hunk 7]
```diff
@@ -39,7 +39,10 @@ nav:
       - Digital Twins: 
           - Create: user/digital-twins/create.md
           - Lifecycle: user/digital-twins/lifecycle.md
-      - Examples: https://github.com/INTO-CPS-Association/DTaaS-examples
+      - Examples:
+          - Overview: user/examples/index.md
+          - Mass Spring Damper: user/examples/mass_spring_damper.md
```
Comment (prasadtalasila) [inline mkdocs.yml:44]: There will be a link for "Fault injection"

File: docs/user/examples/mass_spring_damper.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (astitva1905) [reply docs/user/examples/mass_spring_damper.md:68]: THis refers to the debug.log file generated in workspace/examples/digital_twins/mass-spring-damper. The workspace/examples/data/mass-spring-damper/output directory is also mentioned above it.

File: docs/user/examples/cosimconfig.png
Comment (prasadtalasila) [inline docs/user/examples/cosimconfig.png:1]: You can add this later. Please remove this for now.

**Comment by prasadtalasila** [comment]
@astitva1905 , the changes have been merged via PR #190 . Thanks for the contribution.

**Pull request closed without merging**